bridge/client/app/_components/ktb-webhook-settings/ktb-webhook-settings.component.html

```diff
@@ -28,7 +28,9 @@ <h2>Webhook configuration</h2>
         class="resize-vertical url-input"
       ></textarea>
       <dt-error *ngIf="getFormControl('url').errors?.required">URL must not be empty</dt-error>
-      <dt-error *ngIf="getFormControl('url').errors?.url"> URL must start with http(s):// </dt-error>
+      <dt-error *ngIf="getFormControl('url').errors?.urlOrSecret">
+        URL must start with http(s):// or be a secret
```

Comment:
Not sure if this message can be improved, maybe @johannes-b or @StackScribe can give us feedback?

Reply:
How about: "Must either be a URL (start with <code>http(s)://</code>) or a secret (format: <code>{{...}}</code>)".

The user should somehow be assisted with the format for the secret. This could be added to the placeholder (e.g. "{{.secret.<secretName>.<secretKey>}}"). Also, it's unclear that any part of the URL can be a secret reference (e.g. "https://{{.secret.a.b}}").